src/lando/main/management/commands/start_landing_worker.py

```diff
@@ -46,11 +46,23 @@ def _handle(self, worker: Worker, repo_type: str):
             # Check if any associated repos are unsupported, raise exception if so.
             repo.raise_for_unsupported_repo_scm(repo_type)
             scm = repo.get_scm()
-            scm.prepare_repo(repo.pull_path)
+            repos_to_handle = []
+            try:
+                scm.prepare_repo(repo.pull_path)
+                repos_to_handle.append(repo)
+            except Exception as e:
```

Comment:
What sorts of exceptions are we expecting here? It would be good to be more explicit about them.

Reply:
I saw `ValueError`s. However I don't think we want to be specific now, so we can capture all errors, expected or not, rather than die on unexpected ones.